src/main/resources/application-local.yml

```diff
@@ -4,6 +4,14 @@ notify:
 interventions-ui:
   baseurl: http://localhost:3000
 
+communityapi:
+  baseUrl: http://localhost:8081
+  temporaryReferralRequest:
```

Comment:
Used until we can decide on the default data. Team code made be retrieved from community-api in the future.

Reply:
Could we call it "notification", instead of "referral", please? Maybe `contactNotificationContext`?

It'll be the same for all notification purposes later, not only for referrals.